wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/CoreLoader.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
```

Comment:
super nit: 2020?

Reply:
I put 2019 because this file is extracted from the existing CoreLoader (which has 2019). I don't really know which is right...